puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.8.groovy

```diff
@@ -0,0 +1,109 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Mash Koji Repositories') {
+            agent { label 'sshkey' }
+
+            steps {
+
+                sh "ssh -o 'BatchMode yes' root@koji.katello.org katello-mash-split-3.8.py"
+
+            }
+        }
+        stage('Client Repoclosure') {
```

Comment:
I thought we also had el5 as a client

Reply:
We do, but it's repoclosure is tricky due to it's age and so even if oyu look at nightly we dropped el5 repoclosure.